app/frontend_messages.py

```diff
@@ -0,0 +1,15 @@
+from flask.ext.babel import gettext as _
+
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
+                'INSTRUCTIONS': _("Close this window to continue with your current survey."),
+                },
+            }
+
+
+def get_message(message):
```

Comment:
Zen: Readability counts: is this getting a single message or messages, what is it looking up? i.e. is the function name and parameter clear as to what it is doing and required?